scripts/test.sh

```diff
@@ -59,7 +59,7 @@ start_testrpc() {
   if [ "$COVERAGE" = true ]; then
     node --max-old-space-size=8192 node_modules/.bin/testrpc-sc --gasLimit 0xfffffffff --port "$testrpc_port" "${accounts[@]}" > /dev/null &
   else
-    node_modules/.bin/ganache-cli --gasLimit 8000000 "${accounts[@]}" > /dev/null &
+    node_modules/.bin/ganache-cli --hardfork istanbul --gasLimit 8000000 "${accounts[@]}" > /dev/null &
```

Comment:
Shall we change the gasLimit here to 10,000,000?

Reply:
Same for package.json

Reply:
I want to keep as it is so we can get an alarm

Reply:
I'd vote to set it to at least 9.5M. This way, we have enough headroom and we can recommend higher values for batch functions.